src/Message/FetchTransactionRequest.php

```diff
@@ -11,6 +11,8 @@
      */
     public function getData()
     {
+        $this->validate('transactionReference');
```

Comment:
I'm aware of the `validate` API. But the `getData` API doesn't communicate any exceptions and expects a return value.